public/data/PrasunDas99.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "Prasun Das",
+  "type": "personal",
+  "bio": "SDE @Quantorix Ex-IEMLabs Ex-Aberrantz| Trainer | Web Developer| Microservices | Open Source Contributor | Blockchain | Innovation Enthusiastic",
```

I recommend adding extra spaces here. What do you think? 🙂
```suggestion
  "bio": "SDE @Quantorix Ex-IEMLabs Ex-Aberrantz | Trainer | Web Developer | Microservices | Open Source Contributor | Blockchain | Innovation Enthusiastic",
```